mysql-test/suite/galera/t/galera_replica_no_gtid.cnf

```diff
@@ -0,0 +1,20 @@
+!include ../galera_2nodes_as_slave.cnf
+
+[mysqld]
+wsrep-debug=1
+
+[mysqld.1]
+server_id=15
```

Comment:
It would make sense to keep the common variables in the [mysqld] section.

Reply:
Fixed